build_process/general.md

```diff
@@ -0,0 +1,57 @@
+# General Best Practices
+
+## NPM Scripts
+
+Our projects use different build systems (Gulp, Grunt, custom, etc), but many times they have tasks that do  
+similar things on every project. As developers move from project to project we can save some cognitive overhead by  
+standardizing the commands we use for these common tasks. With node we can use [npm scripts](https://docs.npmjs.com/misc/scripts) as shortcuts to our various common tasks.
+
+- `npm start` - The start script should start up the web server on production or otherwise run and serve a development build
+- `npm test` - Run your tests
+- `npm run` - List all available tasks for this project
+
+### `package.json`
+
+- The default start script runs the `start.js` file which has conditional logic to run either
```

Comment:
I think the wording can be changed to provide clarity. Does this make more sense?

"The default `npm start` command is configured to run the `start.js` file..."